src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/LibreOfficeDocumentConversionService.kt

```diff
@@ -23,80 +22,65 @@ class LibreOfficeDocumentConversionService(
   private val tempDocumentDir: Path,
   private val s3Client: S3Client,
   private val bucketName: String,
+  private val gotenbergWebClient: WebClient,
+
 ) : DocumentConversionService {
 
   override fun convert(multipartFile: MultipartFile): UUID? {
+    // val gotenbergWebClient = WebClient.builder().baseUrl("http://localhost:9091").build()
```

Comment:
Can we cleanup the unused tempDocumentDir stuff?

Reply:
Done